braindecode/models/deepsleepnet.py

```diff
@@ -0,0 +1,239 @@
+import torch
+import torch.nn as nn
+
+
+class small_CNN(nn.Module):  # smaller filter sizes to learn temporal information
```

Comment:
if this class is for internal is should be prefixed with `_` and class names should be in CamelCase so:
```suggestion
class _SmallCNN(nn.Module):  # smaller filter sizes to learn temporal information
```